scripts/run_design.py

```diff
@@ -63,6 +64,7 @@ def main():
     parser.add_argument("--reverse", help="Reverse args", type=str, default="")
     parser.add_argument("--err", help="Error flow args", type=str, default="")
     parser.add_argument("--quiet", action="store_true")
+    parser.add_argument("--experiment_dir", help="Experiment directory", type=str, default=None)
```

Comment:
The `type` should be `pathlib.Path` if the user is providing a path.

Reply:
reverted